README.md

```diff
@@ -163,6 +163,7 @@ Please follow the below steps for Windows:
 
   ```bash
   ctest
+  ./run_tests.py -u
```

Comment:
We do `-u` to update test references. To run test simply do `./run_tests.py`

Reply:
The Linux sections didn't mention `./run_tests.py -u` command anywhere, I want to add that. I spent quite a lot of time in figuring that out.

Reply:
Also the macos test doesn't seem to pass, what can be the reason?

Reply:
> The Linux sections didn't mention ./run_tests.py -u command anywhere, I want to add that.

Then you may add a new subsection below it mentioning `Update test references` and under it `./run_tests.py -u`

Reply:
> what can be the reason?

Looks like some kind of infinite loop ran over, you may need to check if there exists any kind of error.

Reply:
> > what can be the reason?
> 
> Looks like some kind of infinite loop ran over, you may need to check if there exists any kind of error.

In that case, why do other tests pass?

Reply:
> > The Linux sections didn't mention ./run_tests.py -u command anywhere, I want to add that.
> 
> Then you may add a new subsection below it mentioning `Update test references` and under it `./run_tests.py -u`

Done!